packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -293,6 +297,237 @@ QUnit.module('MessageBox', moduleConfig, () => {
         });
     });
 
+    QUnit.module('onTypingStart event', () => {
+        QUnit.test('should be triggered once if a character is entered in the input', function(assert) {
+            const onTypingStartStub = sinon.stub();
+
+            this.reinit({ onTypingStart: onTypingStartStub });
+
+            keyboardMock(this.$input)
+                .focus()
+                .type('n');
+
+            assert.strictEqual(onTypingStartStub.callCount, 1);
+        });
+
+        QUnit.test('should be possible to update it at runtime', function(assert) {
```

Comment:
Also we should be sure that we can update when 'onTypingStart' option already has value

Reply:
Okay, I have added a test: _should be possible to update it at runtime if init value is function_